UnitsNet/CustomCode/UnitSystem.cs

```diff
@@ -27,8 +27,6 @@
 using JetBrains.Annotations;
 using UnitsNet.I18n;
 
-// ReSharper disable once CheckNamespace
```

Comment:
This line should be left untouched, please revert that. It's needed for ReSharper plugin to play nice when doing automated code cleanup, since the file location dictates a different namespace.